app/i18n/en/admin.php

```diff
@@ -14,7 +14,7 @@
 	'auth' => array(
 		'allow_anonymous' => 'Allow anonymous reading of the default user’s articles (%s)',
 		'allow_anonymous_refresh' => 'Allow anonymous refresh of the articles',
-		'api_enabled' => 'Allow <abbr>API</abbr> access <small>(required for mobile apps)</small>',
+		'api_enabled' => 'Allow <abbr>API</abbr> access <small>(required for mobile apps and sharing user queries by link)</small>',
```

Comment:
Is it a good wording?
I am thinking about to set a link behind "user queries" that links to the user queries config page. 
Any comments here?

Reply:
via link, but seems fine otherwise

Reply:
I think *sharing user queries* should be enough, without the *via link* which sounds superfluous to me

Reply:
True enough.

Reply:
> via link, but seems fine otherwise

;) ⬇
![grafik](https://github.com/user-attachments/assets/e3494aba-87eb-4ffa-b962-692e41ed2b8d)

Reply:
Fair point, although *share this query with anyone* does not have the *by link*.
*by HTML* is different as it is a distinction